pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -123,6 +123,11 @@ message MessageMetadata {
     // internal metadata instead of application published data.
     // Markers will generally not be propagated back to clients
     optional int32 marker_type = 20;
+
+	// transaction related message info
+	optional int64 indirect_ledger_id = 21;
```

Comment:
I don't think we need this field anymore. We can remove it for now. If we really need it in future, we can add it back.

Reply:
Sure.